compiler/luci/partition/include/luci/Partition.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PARTITION_H__
+#define __LUCI_PARTITION_H__
+
+#include <luci/IR/Module.h>
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace luci
+{
+
+/**
+ * @brief PartitionTable holds partition information
+ */
+struct PartitionTable
+{
+  std::vector<std::string> groups;
+  std::string def_group;
```

Comment:
What is difference between `group` and `def_group`? Maybe the one who has made the `PartitionTable`?

Reply:
`group` as an assigned group for particular OPCODE (of some Node(s))
`def_group` is a group for not assigned, a default group.

Reply:
`PartitionTable` source is, for now, INI format file, wrote by the user who want's to partition the circle file.
`PartitionTable` content will reflect what the user gave.

Reply:
Oh, I got it with example INI file.
```INI
[partition]
backends=cpu,acl_cl
default=cpu
comply=opcode

[OPCODE]
SUB=acl_cl
```
`[partition]` -> `PartitionTable`
`backends` -> `groups`
`default` -> `def_group`
`[OPCODE]` -> byopcodes

Reply:
`backends` is translated to `group` for general term in luci_partition ...

Reply:
(Optional) Can we change `def_group` to `default_group` ?
I thought that `def` is `def` of `def/use` :(

Reply:
> Can we change def_group to default_group ?

OK :)